driver/ppm_flag_helpers.h

```diff
@@ -1573,6 +1580,18 @@ static __always_inline u32 execveat_flags_to_scap(unsigned long flags)
 	return res;
 }
 
+static __always_inline u32 fsconfig_cmds_to_scap(u32 cmd)
+{
+	/*
```

Comment:
This concerns me a bit. Since we support reproducible captures, I would not rely on an external enumeration even if it's supposed to be stable. Given that the additional work here is just an enum, I would provide an internal representation (even if reduntant). Just my 2 cents.

Reply:
The internal representation is there, there is just no need to map between one and the other because it is an enum in userspace API for kernel too ;) so it is stable and arch independent.